src/pages/index.js

```diff
@@ -6,121 +6,415 @@ import Section from "../components/Section.js";
 import PopupWithImage from "../components/PopupWithImage.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import UserInfo from "../components/userInfo.js";
+import Api from "../components/Api.js";
 
 import "../pages/index.css";
+import Popup from "../components/Popup.js";
+import PopupWithDeleteConfirmation from "../components/PopupWithDeleteConfirmation.js";
 
 // CONSTANT
+const deleteCardButton = document.querySelector(".cards__delete-button");
+const confirmDeleteButton = document.querySelector("#confirm-delete-button");
 const addNewCardButton = document.querySelector("#profile-add-button");
 
+
+
+
+
+
 const profileEditButton = document.querySelector("#profile-edit-button"); //storing edit button inside this variable
 const profileTitle = document.querySelector(".profile__title "); //target id on HTML and create variable on JS
 const profileDescription = document.querySelector(".profile__description");
 const profileTitleInput = document.querySelector("#profile-title-input");
 const profileDescriptionInput = document.querySelector(
   "#profile-description-input"
 );
-//const modal = document.querySelector(".modal");
+
 
 const cardsListItem = document.querySelector(".cards__list-item");
 const modalImage = document.querySelector(".modal__image");
 const cardTitleInput = document.querySelector("#card-title-input");
 const cardUrlInput = document.querySelector("#card-url-input");
 
+const profileAvatarElement = document.querySelector(".profile__avatar");
+const profileAvatarUrlInput = document.querySelector("#avatar-url-input");
+const profileAvatarButton = document.querySelector(".profile__container-edit-icon");
+
 const editProfileModal = document.querySelector("#profile-edit-modal");
-const editProfileForm = editProfileModal.querySelector(".modal__form_profile");
+const editProfileForm = editProfileModal.querySelector("#edit-profile-form");
+const editProfileSubmitButton = editProfileModal.querySelector(".modal__button");
 
-const addNewCardModal = document.querySelector("#add-card-modal");
+//const updateAvatarModal = document.querySelector("#update-avatar-modal");
+const updateAvatarForm = document.querySelector("#update-avatar-form");
```

Comment:
**Could be improved**

We can get any form using their **unique** attribute `name` or `id` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

You need to add `name` or `id` first in `index.html` to achieve that.

More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms